reconstructSPI/iterative_refinement/expectation_maximization.py

```diff
@@ -315,7 +317,7 @@ def grid_SO3_uniform(n_rotations):
 
     @staticmethod
     def generate_xy_plane(n_pix):
-        """Generate xy plane.
+        """Generate (x,y,0) plane centered about (0,0,0).
```

Comment:
Put in details about how the left half and right half are set up. It's a constant source of confusion in development and important to be mentioned explicitly. It connects to conventions in the discrete Fourier transform.

It comes directly from the return of np.arange. np.arange(4) returns (0,1,2,3) and np.arange(-2,2) returns (-2,1,0,1). In general (-n//2,..., -1, 0, ..., n//2-1) with the left half ending at -1 and the right half starting at 0.

Also in our code base we assume that n_pix is always even. My bet is that things would break if not. So that should be mentioned here.